src/Symfony/Component/Messenger/Stamp/HandledStamp.php

```diff
@@ -77,11 +94,17 @@ public function getResult()
         return $this->result;
     }
 
+    /**
+     * @return string
```

Comment:
Please remove this phpdoc. We don't add it when it provides no value compared to the signature. Here, it only duplicates the return type.

Reply:
actually, this applies to most places changed in that PR.